Example/LogMoji/ViewController.swift

```diff
@@ -9,6 +9,20 @@
 import UIKit
 import LogMoji
 
+enum Errors: String, State {
```

Comment:
I feel like this is counterproductive to call this `Errors` as these are not all errors! Can we call this something along the lines of `States` or `LogStates`? Whatever you think is better here!